build-in-docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/sh
+
+set -e
+
+mkdir -p /catalogue-data/htmlcache
+mkdir -p /catalogue-data/lucene
+
+unzip -o /target/$CATALOGUE.war -d /var/lib/jetty/webapps/$CATALOGUE
```

Comment:
This doesn't exist. I guess you can unzip the WAR in the `build` stage and copy the extracted dir in the next stage. That will save some start time.